CMakeLists.txt

```diff
@@ -19,6 +19,8 @@ cmake_minimum_required(VERSION 3.17.2)
 
 project(VEL LANGUAGES CXX)
 
+option(BUILD_TESTS "Build tests")
+
```

Comment:
Why was this changed?

Reply:
That was to be sure that the option exist because it's used.... but maybe it's not how conceptionally it works. 
`BUILD_TESTS` now used when lunarg_target_compiler_configurations is called.

Reply:
Why not use `BUILD_WERROR`?

Reply:
Oups, mistake!

Reply:
Fixed!